generators/spa/templates/VueJS/common/packages.config

```diff
@@ -1,11 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="DotNetNuke.Core" version="9.8.0" targetFramework="net472" />
-  <package id="DotNetNuke.Web" version="9.8.0" targetFramework="net472" />
-  <package id="DotNetNuke.WebApi" version="9.8.0" targetFramework="net472" />
+  <package id="DotNetNuke.Core" version="9.11.0" targetFramework="net472" />
```

Comment:
@ArielBlanco1990 There should be a variable to use for this, instead of hard-coding the version number here. Please follow the pattern that was in place before this update. It previously used `dnnBuildVersionShort`. If that doesn't work and there's not a `dnnBuildVersionLong` or something, then we should have it created. Ideally, this update should be done in as few places as possible when we need to do it again in the future.